libs/form-configuration/core/README.md

```diff
@@ -0,0 +1,53 @@
+# @nrich/form-configuration-core
+
+## Overview
+
+`@nrich/form-configuration-core` is a modules for generating automatic validations for forms in the application.
+It's a frontend part of [nrich-form-configuration](https://github.com/croz-ltd/nrich/tree/master/nrich-form-configuration) backend module.
+Together, they allow the user to define validations on a single place (backend).
+
+For validation schemas this lib uses [yup](https://github.com/jquense/yup).
+
+## Setup
+
+To use this module in your project run `npm install @nrich/form-configuration-core` or `yarn add @nrich/form-configuration-core`
+
+## Usage
+
+On some upper level of your app, wrap your components in `FormConfigurationProvider`.
+
+```tsx
+import { FormConfigurationProvider } from "@nrich/form-configuration-core";
+
+const App = () => (
+  <FormConfigurationProvider loader="Loading..." url="/nrich/form/configuration">
+    {/* rest of the app... */}
+  </FormConfigurationProvider>
+);
+```
```

Comment:
Should we maybe include a bit of desription here about parameters that can be passed to FormConfigurationProvider? Or is the description inside the component enough?

Reply:
Added!